vNext/test/Tester/Tester.csproj

```diff
@@ -57,6 +57,39 @@
       <Project>{40ee3b00-d381-485f-9c69-ff706837deed}</Project>
       <Name>OrleansTestingHost</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\src\Orleans\Orleans.csproj">
```

Comment:
lol, now I see #2323 (which wasn't there when I started reviewing).
Still, surprised that Tester project is there in vNext (confirmed by looking at master)... It is still empty though